src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedRespondentNotificationHandler.java

```diff
@@ -26,11 +28,15 @@
 
     private static final List<CaseEvent> EVENTS = List.of(
         NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIMANT_CONFIRMS_TO_PROCEED,
+        NOTIFY_RESPONDENT_SOLICITOR2_FOR_CLAIMANT_CONFIRMS_TO_PROCEED,
         NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIMANT_CONFIRMS_TO_PROCEED_CC);
 
     public static final String TASK_ID = "ClaimantConfirmsToProceedNotifyRespondentSolicitor1";
+    public static final String Task_ID_RespondentSol2 = "ClaimantConfirmsToProceedNotifyRespondentSolicitor2";
```

Comment:
Similar to my comment above, please can we uppercase and name them TASK_ID_RESPONDENTSOL1 and TASK_ID_RESPONDENTSOL2